Src/AutoFakeItEasy.FakeItEasy2UnitTest/AutoFakeItEasy.FakeItEasy2UnitTest.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" Version="2.0.0" />
+    <PackageReference Include="FakeItEasy" Version="[2.0.0]" />
```

Comment:
Wouldn't it be better to use "[2,3)" or "[2,2.1)" if you want only patch revisions to be let through.

Reply:
Logically it probably makes sense, however I prefer the strict specification. I want to test the initial version only, as SemVer guarantees that if the initial version is OK - any minor will be OK too. Actually, even the previous specification worked as it always resolved to the lower range version, but I wanted to be double sure and therefore altered it.

Given that this is a test project and no other project depends on it, I'd prefer to keep it as is.

Reply:
I see your point, but what if I need a non-breaking feature introduced in the 2.1.0?

Reply:
Where do you need that feature? If you look more precisely, you will find that this project's folder is totally empty and project contains no own files. Rather, it imports files from other test project. Its the only purpose is to run our existing unit tests under the different test framework and verify that we haven't broken them for a newer supported version. This project is _never_  intended to have own files so it cannot use features from this particular version. Therefore, your point seems to be not applicable here, or I'm missing something. :confused:

Reply:
Ouch, my bad :)
I was on mobile when I added this comment and I didn't see that you implemented the "[x,y)" approach in other projects. I had wrongly assumed that this was the approach for the whole PR.
Apologies.

Reply:
No worries! Thanks for bringing an extra-attention on these lines and thanks for monitoring our PRs - appreciate that. That's really cool! If I make a wrong decision (or simply overlooked something) you might save the world 😉

I'm happy that this time everything was fine.